classes/submission/Submission.inc.php

```diff
@@ -93,8 +93,7 @@ public function _getContextLicenseFieldValue($locale, $field, $publication = nul
                         case 'issue':
                             if ($publication->getData('issueId')) {
                                 // override to the issue's year if published as issue-based
-                                $issueDao = & DAORegistry::getDAO('IssueDAO');
-                                $issue = $issueDao->getBySubmissionId($this->getId());
+                                $issue = Repo::issue()->getBySubmissionId($this->getId());
```

Comment:
I can see you're reproducing the way it was done before, but it doesn't make a lot of sense to go back to the submission via `getBySubmissionId(...)`. We've got the issue ID already in `$publication->getData('issueId')` so we can pull it out with `Repo::issue()->get(...)` directly.